metafacture-biblio/src/main/java/org/metafacture/biblio/marc21/Marc21Encoder.java

```diff
@@ -170,7 +170,7 @@ public void startEntity(final String name) {
 
     private void startField(final String name) {
         if (name.length() != NAME_LENGTH) {
-            throw new FormatException("invalid entity name: " + name);
+            throw new FormatException("invalid leader entity name: " + name);
```

Comment:
No, that's not (necessarily) the leader - it can be any entity.

Reply:
```suggestion
            throw new FormatException("invalid entity name: " + name);
```